packages/card/src/Card.tsx

```diff
@@ -21,7 +21,7 @@ export interface CardProps extends ComponentProps<'div'> {
 export const Card = forwardRef(function Card(
   {
     className,
-    ratio = 'rectangle',
+    ratio = 'custom',
```

Comment:
```suggestion
    ratio = 'rectangle',
```
기본값이 변하는건 breaking changes라서 메이저 버전업이 되어야할텐데, 특별한 이유가 없다면 기존과 동일하게 유지하면 어떨까?

Reply:
아.. 그렇다면 일단 유지시킬게 ~